upcoming-release-notes/6042.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Bugfix
+authors: [Jayant-Jeet]
+---
+
+Include the current week in Net Worth report when viewing by Weekly interval. Fixes [#5986](https://github.com/actualbudget/actual/issues/5986)
```

Comment:
Please could you remove the link to the issue? It doesn't need to be in the published notes when they're collected on release.

```suggestion
Include the current week in Net Worth report when viewing by Weekly interval.
```